### PR TITLE
Fix markdown syntax

### PR DESCRIPTION
**Pull request by lpil**

### CODE REVIEW

**Review by chouzar — APPROVED** [review]

File: docs/ABOUT.md [hunk 1]
```diff
@@ -2,23 +2,23 @@
 
 [Gleam](https://gleam.run/) is designed with the followinng principles in mind:
```
Comment (jiegillet) [inline docs/ABOUT.md:3]:
```suggestion
[Gleam](https://gleam.run/) is designed with the following principles in mind:
```

File: docs/ABOUT.md [hunk 2]
```diff
@@ -2,23 +2,23 @@
...
+## Be safe
 
 An expressive type system inspired by the ML family of languages helps us find and prevent bugs at compile time, long before it reaches your users.
 
 For the problems the type system can't solve (such as your server being hit by a bolt of lightning) the [Erlang/OTP](https://learnyousomeerlang.com/what-is-otp) runtime provides well tested mechanisms for gracefully handling failure.
 
-##Be friendly
+## Be friendly
 
 Hunting down bugs can be stressful so feedback from the compiler should be as clear and helpful as possible. We want to spend more time working on our application and less time looking for typos or deciphering cryptic error messages.
```
Comment (jiegillet) [inline docs/ABOUT.md:13]:
```suggestion
Hunting down bugs can be stressful so feedback from the compiler should be as clear and helpful as possible.
We want to spend more time working on our application and less time looking for typos or deciphering cryptic error messages.
```

File: docs/ABOUT.md [hunk 3]
```diff
@@ -2,23 +2,23 @@
...
 
 As a community we want to be friendly too. People of all backgrounds, genders, and experience levels are welcome and must receive equal respect.
```
Comment (jiegillet) [inline docs/ABOUT.md:15]:
```suggestion
As a community we want to be friendly too.
People of all backgrounds, genders, and experience levels are welcome and must receive equal respect.
```

File: docs/ABOUT.md [hunk 4]
```diff
@@ -2,23 +2,23 @@
...
 
-##Be performant
+## Be performant
 
 The Erlang/OTP runtime is known for its speed and ability to scale, enabling organisations such as WhatsApp and Ericsson to reliably handle massive amounts of traffic at low latency. Gleam should take full advantage of this runtime and be as fast as other BEAM languages such as Erlang and Elixir.
```
Comment (jiegillet) [inline docs/ABOUT.md:19]:
```suggestion
The Erlang/OTP runtime is known for its speed and ability to scale, enabling organisations such as WhatsApp and Ericsson to reliably handle massive amounts of traffic at low latency.
Gleam should take full advantage of this runtime and be as fast as other BEAM languages such as Erlang and Elixir.
```

**Review by jiegillet — COMMENTED** [review]
Tiny fixes :)

**Comment by jiegillet** [comment]
lol, one minute too late!

**Comment by lpil** [comment]
Oops 😁

**Pull request merged**